### DISCUSSION TITLE
Can I start a rqlite database inside my app code?

### DISCUSSION THREAD

**Discussion by pyh4** [rqlite/rqlite#1142] [Q&A] [answered]
Can I start a rqlite database inside my app code, like what can be done with go-sqlite?

```go
package main

import (
	"database/sql"
	"log"

	_ "github.com/glebarez/go-sqlite"
)

func main() {
	db, err := sql.Open("sqlite", ":memory:")    // a sqlite database is started here.
	if err != nil {
		log.Fatal(err)
	}

	_ = db.QueryRow("select sqlite_version()")
}
```

Thank you!

**Comment by otoolep** [comment, marked as answer]
We do not (yet) have a Go SQL driver for rqlite.

Instead you must write your own HTTP-based client or just use the existing client library at https://github.com/rqlite/gorqlite
